### ISSUE TITLE
Last match, result, and rewards from events missing

### ISSUE THREAD

**Issue by discomethod** [Razviar/mtga-pro-tracker#481]
**Describe the bug**
Win-loss records and rewards for draft events are not showing up correctly on my events page and daily summary.

**To Reproduce**
1. Go to my events page at https://mtgarena.pro/events/#discomethod.
2. The results for two drafts on August 7 are both 6:2, even though both events were played to completion. No rewards are listed for either event.
3. Likewise, on the daily summary for my feed, the win-loss records for the two draft decks I played are incorrect.

**Expected behavior**
The Gruul deck ended 6:3 without receiving the bonus pack prize, and the Temur deck ended 7:2. The records are still incorrect after performing manual resync.

**Screenshots**
![2019-08-08 mtgap bug](https://user-images.githubusercontent.com/3166008/62743741-98116d00-ba11-11e9-8551-6e372fbef2e5.PNG)
![2019-08-08 mtgap bug 2](https://user-images.githubusercontent.com/3166008/62743742-9a73c700-ba11-11e9-8de0-64c0ca19146a.PNG)

**Additional context**
Seems like the last match I played for each of these events couldn't be parsed, which could explain why the rewards and result of the final game are what's missing.

**Comment by Aquaman88** [comment]
Thank you! We will investigate your issue! Do you have this issue during the 8th of August too?

**Comment by discomethod** [comment]
@Aquaman88 Yes, in fact I've played a few more drafts since and they seem to be fully correct (on daily feed and events page).

**Comment by Aquaman88** [comment]
Thank you for myour information. Unfortunately, the information about matches sometimes can be missed. There are a lot of reasons of this phenomenon but the main reason of it situation can be a lot of synchronous processes. We can't debug all of them, but we are still trying to find the solution. We cannot guarantee any terms of fixing, so we have to deal with some missing matches.
